_posts/en/newsletters/2024-08-09-newsletter.md

```diff
@@ -0,0 +1,476 @@
+  Robin Linus announced [Dark Skippy][], an improved method for key
+  exfiltration from a Bitcoin signing device which they previously
+  [responsibly disclosed][topic responsible disclosures] to
+  approximately 15 different hardware signing device vendors.  _Key exfiltration_ occurs
+  when transaction signing code deliberately creates its signatures in
+  such a way that they leak information about the underlying key
+  material, such as a private key or a [BIP32 HD wallet seed][topic
+  bip32].  Once an attacker obtains a user's seed, they can steal any of
+  the user's funds at any time (including funds spent in the transaction
+  that results in exfiltration, if the attacker acts quickly).
+
+  The authors mention that the best previous key exfiltration attack of
+  which they are aware required dozens of signatures to exfiltrate a
+  BIP32 seed.  With Dark Skippy, they are now able to exfiltrate a seed
+  in two signatures, which may both belong to a single transaction with
+  two inputs, meaning all of a user's funds may be vulnerable from the
+  moment they first try to spend any of their money.
+
+  Key exfiltration can be used by any logic that creates signatures,
+  including software wallets---however, it's generally expected that a
+  software wallet with malicious code will simply transmit its seed to
+  the attacker over the internet.  Exfiltration is mainly considered to
+  be a risk for hardware signing devices that don't have direct access
+  to the internet.  A device that has had its logic corrupted, either
+  through its firmware or through hardware logic, can now quickly exfiltrate a
+  seed even if the device is never connected to a computer (e.g. all
+  data is transferred using NFC, SD cards, or QR codes).
+
+  Methods of performing [exfiltration-resistant signing][topic
+  exfiltration-resistant signing] for Bitcoin have been discussed
+  (including in Optech newsletters as far back as [Newsletter
+  #87][news87 anti-exfil]) and are currently implemented in two hardware
+  signing devices that we are aware of (see [Newsletter #136][news136
+  anti-exfil]).  That deployed method does require an extra round trip
+  of communication with the hardware signing device compared to standard
+  single-sig signing, although that may be less of a downside if users
+  become accustomed to other types of signing, such as [scriptless
+  multisignatures][topic multisignature], that also require additional
+  round trips of communication.  Alternative methods of exfiltration-resistant
+  signing that offer different tradeoffs are also known,
+  although none has been implemented in Bitcoin hardware signing devices
+  to our knowledge.
+
+  Optech recommends that anyone using hardware signing devices to protect
+  substantial amounts of money guard against corrupted
+  hardware or firmware, either through the use of exfiltration-resistant
+  signing or through the use of multiple independent devices (e.g. with
```

Comment:
```suggestion
  signing or multiple independent devices (e.g. with
```

Reply:
Leaving as-is.  This puts the verb ("use") too far from the second term ("multiple independent devices").